{"failedTests":[{"testName":"parallel/test-async-context-frame","timestamp":1730450974630,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win11-COMPILED_BY-vs2022 #31099","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022/31099/","buildNumber":31099,"builtOn":"test-azure_msft-win11_vs2022-x64-2","nodeVersion":"24.0.0","commitHash":"efe96fc76b83565468f28c4a42522e44bf4d9624","callStack":[{"upstreamBuild":66806,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":75552,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":63376,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"55623"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":285,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (998.9372ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (930.4015ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (1003.5137ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (1094.0137ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (842.6626ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (1164.1531ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (1005.746ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (801.3777ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (1100.1615ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (911.6485ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (926.1663ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (914.1284ms)\n  ✖ async-hooks\\test-async-local-storage-no-mix-contexts.js (1219.5992ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-no-mix-contexts.js failed with exit code 3221225477\n    \n    3221225477 !== 0\n    \n        at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:935:9)\n        at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-promises.js (949.1134ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (906.8001ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (735.47ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (1069.555ms)\n  ✔ parallel\\test-async-local-storage-bind.js (752.7747ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (732.8633ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (767.4464ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (781.6963ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (1374.928ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (986.4099ms)\n✖ AsyncContextFrame (21977.9163ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 21984.7735\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:49:5\n✖ async-hooks\\test-async-local-storage-no-mix-contexts.js (1219.5992ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-no-mix-contexts.js failed with exit code 3221225477\n  \n  3221225477 !== 0\n  \n      at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:935:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":22393.255,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}